protocol/avdl/keybase1/teams.avdl

```diff
@@ -619,26 +618,41 @@ protocol teams {
   @typedef("string")
   record TeamInviteName {}
 
+  // For Keybase, Seitan, Invitelink, etc. invites, we want to show a different
+  // invite name than what is in the sigchain (which may be an encoded
+  // userversion or a messagepack blob, e.g.)
+  @typedef("string")
+  record TeamInviteDisplayName {}
```

Comment:
Clear separation of concerns from TeamInviteName which is a implementation detail more or less.

e.g. Display name will be formatted url, or max@twitter, etc whereas the name itself may be messagepacked pkey, uv percent form, whatever.